packages/dalek/lib/main.js

```diff
@@ -0,0 +1,19 @@
+/** @babel */
+
+const dalek = require('./dalek')
+const Grim = require('grim')
+
+module.exports = {
+  activate () {
+    atom.packages.onDidActivateInitialPackages(async () => {
+      const duplicates = await dalek.enumerate()
+      for (let i = 0; i < duplicates.length; i++) {
+        const duplicate = duplicates[i]
+        Grim.deprecate(
+          `You have the core package "${duplicate}" installed as a community package. See https://github.com/atom/dalek for how this causes problems and instructions on how to correct the situation.`,
```

Comment:
This link is there to show the users the steps in the readme so they can resolve the issue. Should it be changed to the new location of the readme file?

Reply:
Yep, thanks for catching that!

Reply:
Nice catch, @Ben3eeE. :zap: